exercises/Makefile

```diff
@@ -0,0 +1,140 @@
+# Colors
+COLOR=\033[0;32m
+NC=\033[0m
+BOLD=\033[1m
+
+# Vars
+DOCKER_COMPOSE_CMD  = docker-compose -f docker-compose.yml
+KUBECTL_CONFIG_PATH = .output/kubeconfig.yaml
+KUBECTL_CMD         = kubectl --kubeconfig=${KUBECTL_CONFIG_PATH}
```

Comment:
We don't know what version of kubectl is installed on their laptop. If it is a 1.10 or 1.11 for example it will conflict with the k3s version. I'd prefer to use either workshopctl or ./tmp/kubectl as we have ownership on these files and can control versions to avoid a mismatch.

Reply:
Since we're going to maintain workshopctl, and add any required logic inside it, I'd rather use workshopctl for that too.

Reply:
note: we wont have any `kubectl` completion when we use `workshopctl`

Reply:
Fixed: https://github.com/hugoShaka/k8s-workshop/commit/d99f0f3e87262a31d5c4340f194ae3c8e50856ea
now using the workshopctl instead of kubectl directly